nautobot/core/api/utils.py

```diff
@@ -178,6 +178,10 @@ def get_view_name(view, suffix=None):
 
     else:
         # Replicate DRF's built-in behavior.
+        name = getattr(view, "name", None)
+        if name is not None:
+            return view.name
```

Comment:
Is there a reason this is not:

```py
if getattr(view, "name", None) is not None:
    return view.name
```

OR

```py
name = getattr(view, "name", None)
if name is not None:
    return name
```

Just looks weird to me as written?

Reply:
Should be `return name` - transcription error on my part.